runtime/doc/lua.txt

```diff
@@ -124,8 +124,7 @@ Note:
 ==============================================================================
 Lua Syntax Information                                       *lua-syntax-help*
 
-While Lua has a simple syntax, there are a few things to understand,
-particularly when looking at the documentation above.
```

Comment:
What documentation above? This wasn't very clear to me. I checked the original commit to see if there was a documentation that was removed from the file, but it doesn't seem so: https://github.com/neovim/neovim/commit/b39edb5b605b545843482c70a3a3d03e2331a059